subgraph/bin/networks.js

```diff
@@ -46,14 +48,53 @@ const generateNetworksFile = async () => {
   console.log(`Networks file saved at: ${networkFilePath}`)
 }
 
+function setupFolderConfig() {
+  // make sure we clean up
+  if (fs.pathExistsSync(configFolderPath)) {
+    fs.rmSync(configFolderPath, { recursive: true, force: true })
+  }
+  fs.mkdirSync(configFolderPath)
+}
+
+function createNetworkConfig(network, chainName) {
+  const networkName = chainName
+  const networkFile = {
+    network: network.subgraph.networkName,
+    unlock: network.unlockAddress,
+    unlockStartBlock: network.startBlock,
+    unlock2: ethers.constants.AddressZero,
```

Comment:
I wonder if we should not rather use some kind of array here? What happens when/if there is a 3rd deployment of Unlock on a specific network?

Reply:
Good idea, I have implemented a deployments array which will deal with any amount of deployments.